nightfall-deployer/migrations/2_deploy_upgradeable.js

```diff
@@ -44,7 +44,7 @@ module.exports = async function (deployer) {
 
   await deployProxy(Proposers, [], { deployer, unsafeAllowLinkedLibraries: true });
   await deployProxy(Challenges, [], { deployer, unsafeAllowLinkedLibraries: true });
-  await deployProxy(Shield, [], { deployer, unsafeAllowLinkedLibraries: true });
+  await deployProxy(Shield, { deployer, unsafeAllowLinkedLibraries: true });
```

Comment:
Why this change here and not in `Proposers` or `Challenges` about the empty parameters `[ ]`?